src/visuallyCompleteCalculator.ts

```diff
@@ -115,12 +129,22 @@ class VisuallyCompleteCalculator {
       // identify timestamp of last visible change
       const end = Math.max(start, this.lastImageLoadTimestamp, this.lastMutation?.timestamp ?? 0);
 
+      const navigationEntries = performance.getEntriesByType(
+        'navigation'
+      ) as PerformanceNavigationTiming[];
+      const navigationType = isBfCacheRestore
+        ? 'back_forward'
+        : start !== 0
+        ? 'script'
+        : navigationEntries[navigationEntries.length - 1].type;
```

Comment:
I don't think it matters much either way since there is only 1 navigation entry, but maybe `navigationEntries[0]` is easier to read?

Reply:
[This](https://developer.mozilla.org/en-US/docs/Web/API/Performance_API/Navigation_timing) says
> Only the current document is included, so usually there is only one [PerformanceNavigationTiming](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceNavigationTiming) object to observe.

and web-vitals only takes the first navigation entry: https://github.com/GoogleChrome/web-vitals/blob/main/src/lib/getNavigationEntry.ts